packages/utilities/psammead-assets/README.md

```diff
@@ -56,7 +56,7 @@ import { coreIcons } from '@bbc/psammead-assets/svgs';
 
 ## Media Icons SVGs
 
-Media icons is an object containing styled SVG icons for video, audio and photogallery. Media icons are sized to work well alongside specific text with typography group `GEL Minion`. They are used in `psammead-media-indicator` component.
+Media icons is an object containing styled SVG icons for video, audio, photogallery, guidance and clock. Media icons are sized to work well alongside specific text with typography group `GEL Minion`. They are used in `psammead-media-indicator` component.
```

Comment:
Good point! I will add that guidance is used for media-player. But I am a bit unsure about saying that a clock is used for radio schedules since that component has not been developed yet.